### PR TITLE
Websocket dispatch fix

### PR DESCRIPTION
**Pull request by miguelportilla**

### CODE REVIEW

File: src/ripple/websocket/Handler.h [hunk 1]
```diff
@@ -304,8 +304,12 @@ class HandlerImpl
     void message_job(std::string const& name,
                      connection_ptr const& cpClient)
     {
-        auto msgs = [this, cpClient] (Job& j) { do_messages(j, cpClient); };
-        app_.getJobQueue ().addJob (jtCLIENT, "WSClient::" + name, msgs);
+        app_.getJobQueue().postCoro(jtCLIENT, "WSClient",
+            [this, cpClient]
+            (std::shared_ptr<JobCoro> jc)
```
Comment (vinniefalco) [inline src/ripple/websocket/Handler.h]: `auto const& jc`

**Comment by JoelKatz** [comment]
:+1:

**Comment by scottschurr** [comment]
:+1:
I didn't spot any issues.

File: src/ripple/websocket/Connection.h [hunk 2]
```diff
@@ -241,18 +241,6 @@ typename WebSocket::MessagePtr ConnectionImpl <WebSocket>::getMessage ()
 }
 
 template <class WebSocket>
-void ConnectionImpl <WebSocket>::returnMessage (message_ptr const& ptr)
```
Comment (ximinez) [inline src/ripple/websocket/Connection.h]: Need to remove the declaration at line 99.

**Pull request closed without merging**